unfoldingWord/translationCore
# Merge Conflict Fixes

#### This pull request addresses:
This PR updates the merge conflict methods to work with the new USFM 3 workflow.
This PR also address the warnings in the console.


#### How to test this pull request:
To test this PR make sure the project https://git.door43.org/EllDoubleYew/ar_eph_text_ulb works well.
Also test project 
[bhadrawahi_tit.zip](https://github.com/unfoldingWord-dev/translationCore/files/1360208/bhadrawahi_tit.zip)


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2853)
